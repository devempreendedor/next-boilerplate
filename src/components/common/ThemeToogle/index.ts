export { default as ThemeToogle } from './ThemeToogle'
