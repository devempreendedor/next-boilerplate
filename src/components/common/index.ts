export * from './ThemeToogle'
export * from './Layout'
