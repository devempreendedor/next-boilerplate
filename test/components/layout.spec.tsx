import { render, screen } from '../test-utils'
import { Layout } from '@components/common'

describe('Layout Component', () => {
  it('Should render the Layout', () => {
    render(
      <Layout>
        <h1>Layout</h1>
      </Layout>
    )

    const layout = screen.getByText(/Layout/i)

    expect(layout).toBeInTheDocument()
    expect(layout).toMatchSnapshot()
  })
})
