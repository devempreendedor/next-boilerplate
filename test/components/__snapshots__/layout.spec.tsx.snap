// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Component Should render the Layout 1`] = `
<h1>
  Layout
</h1>
`;
